pkg/authorization/api/types.go

```diff
@@ -105,6 +105,7 @@ type RoleBinding struct {
 	RoleRef kapi.ObjectReference
 }
 
+// +genclient=true
```

Comment:
Make sure to check all the types are generated correctly. Although the tag should work one for a file, for some reasons for ScheduledJobs (while adding them to k8s) I had to specify once again the tag right before ScheduledJobs types in internal types.go.